doc/cursors.rst

```diff
@@ -3,16 +3,435 @@
 Cursors
 =======
 
-.. admonition:: Work in progress
-   :class: danger
+Built upon :ref:`lenses`, cursors are a great way to bridge
+value-oriented designs and object-oriented systems. It also allows
+modularity.
```

Comment:
"It enables modularity by allowing one to write components that reference mutable and observable data abstracting away the shape of the data storage."